WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -864,10 +866,9 @@ private extension ProductsViewController {
     func checkBlazeBannerVisibility() {
         viewModel.$shouldShowBlazeBanner
             .removeDuplicates()
-            .combineLatest($isEmpty.removeDuplicates())
```

Comment:
non-blocking 💭 : I wonder we can `combineLatest` with the state for whether the error top banner is shown

Reply:
Updated in d6f6e7180715839c24c3e88e63b5fd57c60ece55 to make `hasErrorLoadingData` a publisher so that we can observe here.